bitcoind-regtest/test/Bitcoin/Core/Test/Generator.hs

```diff
@@ -7,6 +7,7 @@ module Bitcoin.Core.Test.Generator (
 import Bitcoin.Core.RPC (BitcoindClient)
 import qualified Bitcoin.Core.RPC as RPC
 import Bitcoin.Core.Regtest (
+    GeneratorConfig (..),
```

Comment:
we used the data constructor in https://github.com/bitnomial/bitcoind-rpc/pull/14/files#diff-7ff6dd07aefafbe4e8b64081e9618d70814f64ae221bce1b6daa98b610207839R33 but it wasn't imported